pandas/io/formats/style.py

```diff
@@ -1003,19 +1003,23 @@ def hide_columns(self, subset) -> "Styler":
     def _highlight_null(v, null_color: str) -> str:
         return f"background-color: {null_color}" if pd.isna(v) else ""
 
-    def highlight_null(self, null_color: str = "red") -> "Styler":
+    def highlight_null(self, null_color: str = "red", subset=None) -> "Styler":
```

Comment:
```suggestion
    def highlight_null(self, null_color: str = "red", subset: Optional[Union[Label, Sequence[Label]]]=None) -> "Styler":
```

Reply:
done, but not sure why 1 check failed.

Reply:
Can you merge master and re-push? CI has been flakey past 2 days but should be fixed now